docs/cli/README.md

```diff
@@ -0,0 +1,13 @@
+# nais-cli
+
+nais-cli is a simple CLI application that developers in NAV can use.
+
+If you want to know more about the tool, please visit and read
+the [github repository](https://github.com/nais/nais-cli) documentation.
```

Comment:
Syns vi burde ha dokumentasjonen i nais-doc, og heller doc for de som skal utvikle verktøyet i github.

Reply:
Helt enig, vi skal fikse lit på de